applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+By enabling NFTs to hold assets, stake, and interact with other blockchain components similarly to standard accounts,
+NFTAA introduces a novel approach to blockchain interactions.
+
+The core of NFTAA's integration with the Polkadot ecosystem lies in utilizing the Substrate framework to develop custom
+blockchain logic that supports NFT-based accounts.
+These NFT accounts can seamlessly interact with other pallets within the Substrate framework, allowing them to own
+assets, stake, and act as proxies for other accounts.
+This integration takes full advantage of Polkadot’s interoperability features, facilitating seamless communication and
+data sharing across multiple independent blockchains (parachains) within the Polkadot network.
+
+Our motivation for creating this project stems from the desire to address the inherent limitations and inefficiencies
+associated with traditional staking methods.
+Issues such as the illiquidity of staked assets and the complexities involved in transferring ownership pose significant
+challenges.
+By utilizing NFTs as accounts, we aim to provide a more secure and flexible staking solution, enhancing user control and
+ownership.
+This innovative approach opens new avenues for business opportunities and applications in DeFi
+and enterprise blockchain solutions, underscoring our commitment to advancing blockchain technology and its
+applications.
+
+### Project Details
+
+#### Business Overview
+
+NFTs are unique digital assets that can represent a wide range of physical and digital items with
+varying values.
+Traditionally, NFTs operate on the asset side of the owner-asset relationship, lacking the capability to own other
+assets.
+Conversely, accounts represent ownership of assets within the blockchain ecosystem.
+By combining these two concepts, we propose a unique entity that can both be uniquely identifiable and capable of owning
+other assets, leading to the concept of NFTAA.
+
+One of the significant strengths of the Substrate framework is its ability to facilitate the creation of custom
+blockchain logic.
+Through forkless runtime upgrades, it is possible to enhance the functionality of individual modules.
+This flexibility is critical as runtime pallets operate close to the system’s core, offering virtually limitless
+modification options.
+Although the transaction weight is a limiting factor in terms of computing resources available for each transaction, it
+does not detract from the Substrate framework’s overall power and adaptability.
+
+Most Substrate pallets are designed based on a single account architecture, aligning with the NFTAA philosophy and
+facilitating smooth integration with existing infrastructure.
+By treating NFTAAs as conventional accounts, we can easily leverage the existing infrastructure for seamless
+communication.
+Additionally, NFTAAs can function as asset owners or participate in multi-signature schemes, further
+enhancing their utility and security.
```

Comment:
We see this as an opportunity for the security of multi-sig schemes so that NFTAA can change the owner. If we have an ordinary account with a leaked PK there is nothing we can do about it. If we have an NFTAA and the owner of such an account leaks PK it can change the owner (i.e. PK) tied to the NFTAA and thus protect the multi-sig (assuming it makes it before the accountant does). But we acknowledge that this may be controversial and we would probably just write utility instead of utility and security :)

Reply:
thanks, I'd love that. Utility sounds reasonable.